packages/eslint-plugin/ROADMAP.md

```diff
@@ -169,7 +169,7 @@
 | [`number-literal-format`]           | 🛑  | N/A                                                                                 |
 | [`object-literal-key-quotes`]       | 🌟  | [`quote-props`][quote-props]                                                        |
 | [`object-literal-shorthand`]        | 🌟  | [`object-shorthand`][object-shorthand]                                              |
-| [`one-line`]                        | 🛑  | N/A                                                                                 |
+| [`one-line`]                        | 🌟  | [`brace-style`][brace-style]                                                        |
```

Comment:
you need to add the link at the bottom of the document as well

Reply:
Done, thank you…